lib/pbench/agent/tool_meister_start.py

```diff
@@ -882,6 +888,17 @@ def start(_prog: str, cli_params: Namespace) -> int:
                 exc,
             )
             return ReturnCode.EXCCREATETMDIR
+        uuid_file = tm_dir / ".uuid"
+        instance_uuid = str(uuid.uuid4())
+        try:
+            uuid_file.write_text(instance_uuid)
```

Comment:
Should we make this file immutable for other users by running `uuid_file.chmod(644)`? or is it not necessary?

Reply:
There are plenty of ways for people to mess with us if they really want to; that seems like excessive paranoia to me.